libs/game/obstacle.ts

```diff
@@ -35,11 +35,11 @@ namespace sprites {
         }
 
         get x(): number {
-            return this.left + this.width >> 1;
+            return this.left + (this.width >> 1);
```

Comment:
shows how much the old on hit tile was used; this has been giving the location incorrectly since tiles were first implemented -- ``a + b >> c`` === ``(a + b) >> c``